opentracing/tracer.py

```diff
@@ -82,3 +83,24 @@ def close(self):
         fut = Future()
         fut.set_result(True)
         return fut
+
+    def implementation_id(self):
+        """Returns the ImplementationID for this OpenTracing implementation.
+
+        :return: An ImplementationID instance that describes this OpenTracing
+            implementation.
+        """
+        return ImplementationID('noop', '1.0.0')
+
+
+# Per collections.namedtuple, the below defines a class called ImplementationID
+# which can be initialized via positional parameters or a kwargs-style
+# initializer.
+#
+#     dapper_impl = ImplementationID('dapper', '0.1.2')
+#     zipkin_impl = ImplementationID(name='zipkin', version='0.3.4')
+#
+ImplementationID = namedtuple('ImplementationID', 'name version')
+
+# The SemVer (http://semver.org/) of the OpenTracing Python API.
+OPENTRACING_PYTHON_SEMVER = '0.9.0'
```

Comment:
@bcronin do we need anything fancier than this? I think the biggest challenge will be remembering to update it :-/

Of course we can automate that process to a certain extent...